src/core/exporter.js

```diff
@@ -0,0 +1,104 @@
+// module: exporter
```

Comment:
Nit: expand this out a little bit, so it's a bit more clear what this module does (i.e., exports a ReSpec document as either HTML or XML).